ndk/src/font.rs

```diff
@@ -0,0 +1,401 @@
+//! Bindings for [`AFont`], [`AFontMatcher`], and [`ASystemFontIterator`]
+//!
+//! [`AFont`]: https://developer.android.com/ndk/reference/group/font#afont_close
+//! [`AFontMatcher`]: https://developer.android.com/ndk/reference/group/font#afontmatcher_create
+//! [`ASystemFontIterator`]: https://developer.android.com/ndk/reference/group/font#asystemfontiterator_open
```

Comment:
Yes, I linked to `AFont_close` because it is the first function. I can't understand why `AFont` does not have typedefs for the structs. I'll link `AFont` to the page itself. Do you think linking `AFontMatcher` and `ASystemFontIterator` to `_create` or `_open` be useful? Since they're constructors (effectively), I think linking to those functions is useful.